sphinx_doc/build_instructions/build.rst

```diff
@@ -0,0 +1,235 @@
+Build Instructions
+##################
+
+Overview
+********
+
+The instructions to build **Navigation2 Stack** and **Navigation2 Dependencies**.
+
+- Build Options
+  
+  - `Quickstart Using Initial Setup Script`_
+  - `Manually Build From Source`_
+  
+- `Build Troubleshooting Guide`_
+
+**Recommended Platform**
+
+* `Ubuntu 18.04 <http://releases.ubuntu.com/18.04/ubuntu-18.04.3-desktop-amd64.iso>`_
+
+**Build Options**
+
+There two different recommended methods for building Navigation2 stack from source.
+
+**Easy Build**
+
+* An automated method to build Navigation2 in a simple and easy way.
+* To build Navigation2 by using the build script, go to `Quickstart using initial setup script`_
+
+Manually Build From Source
+**************************
+
+Step 1- Build ROS 2
+===================
+
+* `ROS2 Build Instructions <https://index.ros.org/doc/ros2/Installation>`_
+* When building ROS 2 from source, if you want to get the latest version of the ROS 2 packages, make sure that the `ros2.repos` file is from the `master` branch.
+* When installing the ROS 2 dependencies using rosdep, make sure to set `--rosdistro` to the lastest ROS 2 distribution (e.g `eloquent`)
```

Comment:
"lastest" -> "latest"

Reply:
Also I dont know what value this bullet has - later you're going to show a `rosdep` command with `--rosdistro [distribution name]`. That should be abundantly clear to chose their current distribution

Reply:
+1 on this. There's also the `rosdep install` command [here](https://github.com/ros-planning/navigation2/pull/1393/files#diff-9922005eb9744b16e9cfc99d325e62a1R58) and [here](https://github.com/ros-planning/navigation2/pull/1393/files#diff-9922005eb9744b16e9cfc99d325e62a1R82) that defaults to `eloquent`. I think that should either be listed as the latest ros-distro or the specific version being installed.